api/config/config_local.json

```diff
@@ -93,8 +93,6 @@
   "absorb": {
     "externalDepartmentId": "Development",
     "enabledForNewUsers": true,
-    "rtTrainingCourseId": "9ad49c70-3b72-4789-8282-5794efcd4ce1",
-    "ctTrainingCourseId": "3765dc64-cc64-4efa-bfc0-9a4dc2e9d09d",
```

Comment:
We use the same Absorb courses in prod and test. Users are kept separate by being in different Absorb "departments" (the externalDepartmentId config, above) depending on the Workbench environment.